### PR TITLE
Update Gradle wrapper to 6.7

### PR DESCRIPTION
**Pull request by fejd**
Update the Gradle wrapper used by the root and sample
projects from 6.3 to 6.7.

______________________________________
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/master/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

**Comment by robstoll** [comment]
@fejd I guess you have updated the number manually. Use the utility from gradle itself to update the wrapper. At least the jar should change as well.

**Comment by fejd** [comment]
Ah sorry, yes that's how I've done in the past. I'll use `./gradlew wrapper --gradle-version 6.7` instead, and map the distribution type to what's currently in the individual properties files. Not part of this commit but should all projects use the same distribution type, either -bin or -all? I also noticed that the update utility prefers Windows style carriage returns (`^M`) in the .bat files, but the existing files use Linux/Unix line endings. I'll fix that with `dos2unix` to make the change easier to review, but I could make a subsequent commit to fix that if you want. It'd make future upgrades a bit smoother I think.

**Comment by robstoll** [comment]
> but the existing files use Linux/Unix line endings.

maybe worth adding a .gitattributes file declaring clrf for bat files?

Fine for me if you do everything in one commit but two commits are also good. What you think is better :)

File: atrium-scala/gradlew.bat [hunk 1]
```diff
@@ -0,0 +1,89 @@
+@rem
```
Comment (robstoll) [inline atrium-scala/gradlew.bat:1]: I'm sorry, this just changed in master (atrium-scala was renamed to atrium-scala2).
Please rebase your changes on master and re-do the update in atrium-scala2

btw. nice that you have spotted this, forgot to include it in the description :+1:

**Review by robstoll — CHANGES_REQUESTED** [review]

File: samples/maven/.gitattributes [hunk 2]
```diff
@@ -1,6 +0,0 @@
-#
```
Comment (robstoll) [inline samples/maven/.gitattributes:1]: Can you please keep these files in the samples and copy it to the root.
The samples are independent projects where the idea is that you as a user can copy this as a starting point of a new project.

**Pull request closed without merging**